MPF.md

```diff
@@ -792,19 +792,71 @@ allowed values (`LOG_USER`, `LOG_DAEMON`, `LOG_LOCAL0`, `LOG_LOCAL1`,
 
 ### mailto
 
-The address that `cf-execd` should email agent output to.
+The address that `cf-execd` should email agent output to. Defaults to `root@$(default:def.domain)`.
+
+This setting can be customized via Augments, for example:
+
+```json
+{
+  "variables": {
+    "default:def.mailto": {
+        "value": "cfengine-maintainers@example.com",
+        "comment": "Email sent from cf-execd when output differs from the prior execution should be delievered to this distribution group for review"
+    }
+  }
+}
+```
 
 ### mailfrom
 
-The address that output mailed from `cf-execd` should come from.
+The address that output mailed from `cf-execd` should come from. Defaults to `root@$(sys.uqhost).$(def.domain)`.
+
+This setting can be customized via Augments, for example:
+
+```json
+{
+  "variables": {
+    "default:def.mailfrom": {
+        "value": "cfengine@example.com",
+        "comment": "Email sent from cf-execd should come from this address"
+    }
+  }
+}
+```
 
 ### smtpserver
 
-The SMTP server that `cf-execd` should use to send emails.
+The SMTP server that `cf-execd` should use to send emails. Defaults to `localhost`.
+
+This setting can be customized via Augments, for example:
+
+```json
+{
+  "variables": {
+    "default:def.smtpserver": {
+        "value": "cfengine@example.com",
```

Comment:
```suggestion
        "value": "smtp.example.com",
```

And is this assuming a plain SMTP endpoint not encrypted or requiring user information for authentication?

Reply:
yep.